lib/active_admin/csv_builder.rb

```diff
@@ -42,7 +44,7 @@ def column(name, options = {}, &block)
     def build(controller, csv)
       @collection  = controller.send :find_collection, except: :pagination
       columns      = exec_columns controller.view_context
-      options      = ActiveAdmin.application.csv_options.merge self.options
+      options      = @options.clone
```

Comment:
I don't think you need `clone` here?

```suggestion
      options      = self.options
```

should do the trick?